content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -108,32 +119,40 @@ cluster for mesh expansion, run the following commands on a machine with cluster
           -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
     {{< /text >}}
 
-### Setting up the machines
+### Setting up the VM
 
 Next, run the following commands on each machine that you want to add to the mesh:
 
-1.  Install the Debian package with the Envoy sidecar:
+<!-- TODO: find a better way to reference release in the URL instead of manually sync... -->
+
+1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM. For example:
 
     {{< text bash >}}
-    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ dpkg -i istio-sidecar.deb
+    $ export GCE_NAME="your-gce-instance"
+    $ gcloud compute scp --project=${MY_PROJECT} --zone={MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
     {{< /text >}}
 
-1.  Copy the `cluster.env` and `*.pem` files that you created in the previous section to the VM.
+1.  Install the Debian package with the Envoy sidecar.
+
+    {{< text bash >}}
+    $ gcloud compute ssh "${GCE_NAME}"
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.1.0-rc.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
```

Comment:
Got some warning messages, not sure if it is expected or not:

lita@lita-test-vm:~$ sudo dpkg -i istio-sidecar.deb
Selecting previously unselected package istio-sidecar.
(Reading database ... 63810 files and directories currently installed.)
Preparing to unpack istio-sidecar.deb ...
Unpacking istio-sidecar (1.1.0-rc.0) ...
Setting up istio-sidecar (1.1.0-rc.0) ...
Adding group `istio-proxy' (GID 113) ...
Done.
Warning: The home dir /var/lib/istio you specified already exists.
Adding system user `istio-proxy' (UID 109) ...
Adding new user `istio-proxy' (UID 109) with group `istio-proxy' ...
The home directory `/var/lib/istio' already exists.  Not copying from `/etc/skel'.
adduser: Warning: The home directory `/var/lib/istio' does not belong to the user you are currently creating.

Reply:
Not sure. The /var/lib/istio is not supposed to exist on common linux distro. Might because you have installed istio.deb on this particular GCE instane before?